tests/integration/test_distributed_inter_server_secret/test.py

```diff
@@ -13,7 +13,6 @@
 
 def make_instance(name, cfg):
     return cluster.add_instance(name,
-        with_zookeeper=True,
```

Comment:
But why do you think it is useless? There is `ON CLUSTER` queries.

Reply:
Oh.

Reply:
And actually it should just simply configure `clickhouse-keeper`, no zookeeper anyway.